src/bindings/python/src/openvino/frontend/tensorflow/node_decoder.py

```diff
@@ -44,26 +46,47 @@ def tf_attr_to_ov(attr):
     return OVAny(tf_attr_to_numpy(attr))
 
 
+def try_decode(line: bytes):
+    assert isinstance(line, bytes), "Expected bytes, got object of type {}".format(type(line))
+    try:
+        return True, line.decode()
+    except ValueError:
+        return False, line
```

Comment:
why do we need this logic? why can't we decode each string tensor element more universally?

Reply:
This logic is needed to decode bytes to string. Bytes in general can fail to decode, if it is not a string or there is some unknown encoding. For this reason try-catch is used.
If we fail to decode it, then list of bytes objects will be returned from decoder.

What do mean by decoding each string tensor element more universally?

Reply:
In offline discussion it was decided to replace `decode()` with `str()`. Done.